book/quick-brick/quick-brick-project-setup.md

```diff
@@ -0,0 +1,56 @@
+# Get ready to work on Quick-Brick
+
+## iOS/tvOS Quick-Brick
+The following chapter will explain how to create and prepare a project for iOS/tvOS platforms with quick-brick. Please use it when you need to develop or test your iOS/tvOS apps.
+
+### Prerequisites for iOS/tvOS with Quick-Brick
+
+To prepare a project for iOS or tvOS with quick-brick you need to `git clone` the following repositories:
+
+* [ZappAppleBuilder](https://github.com/applicaster/ZappAppleBuilder) - the actual native application that will run the react-native quick-brick bundle.
+* [QuickBrick](https://github.com/applicaster/QuickBrick) - this Repo contains the source code for the Quick Brick Project, which enables to create a Zapp App with a 100% react-native-flavored UI.
```

Comment:
this confuse the developer.
They need the qb personal project (plugin) repo.

Reply:
I have removed that